js/index.js

```diff
@@ -1,7 +1,7 @@
 import "babel-polyfill";
 import Raven from "raven-js";
 import createMiddleware from "raven-for-redux";
-import isButterchurnSupported from "butterchurn/lib/isSupported";
+import isButterchurnSupported from "butterchurn/lib/isSupported.min";
```

Comment:
Webpack should be minifying this for us already. I suppose it won't hurt anything to do it twice, but we don't strictly need to.

Reply:
So I "simplified' by build process for building the library, and now I am only publishing the minified versions to NPM (previously it was the minified version of everything else, but a regular version of isSupported).